hickle/tests/test_hickle.py

```diff
@@ -742,7 +742,7 @@ def test_scalar_compression(test_file_name):
     (Scalars are incompressible!)
     https://github.com/telegraphic/hickle/issues/37
     """
-    data = {'a': 0, 'b': np.float(2), 'c': True}
```

Comment:
Doe the test still pass on `np.float(2)`?

Reply:
Yes, it does but with a deprecation warning from numpy.  The change from np.float(2) to float(2) removed the warning.

Reply:
Want me to yank it?

Reply:
By this change @texadactyl  is fixing is a one miss which went unnoticed by me during preparation of hickle 5 dev.  There i have already fixes several locations triggering this warning but missed at least that one. And yes as it does on all other locations it also dose here now and in future. See the explanation of  @texadactyl  in general discussion